src/match/url-match.ts

```diff
@@ -170,6 +170,9 @@ export class UrlMatch extends Match {
     getAnchorHref() {
         let url = this.getUrl();
 
+        //Strip malicious Unicode SNYK-AUTOLINKER-2438289
+        url.replace('\u202E', '');
```

Comment:
Also, I suggest Strip the most RTLO common `U+202E` and `%E2%80%AE` beside `\u202E`

Reply:
would that be immune to ReDoS?

https://en.wikipedia.org/wiki/ReDoS

Reply:
Hey, not sure how this would actually fix the issue because the code isn't re-assigning the replacement result back to `url`. The code would need to be:

```ts
url = url.replace(/\u202E/g, '');
```

Does your new test pass even without this line?